src/Components/HouseholdDataBlock/HouseholdDataBlock.js

```diff
@@ -104,24 +107,23 @@ const HouseholdDataBlock = ({ handleHouseholdDataSubmit }) => {
     const ageTextfieldProps = {
       inputType: 'text',
       inputName: 'age',
-      inputValue: householdData.age,
+      inputValue: memberData.age,
       inputLabel: createFMInputLabel(personIndex),
       inputError: householdMemberAgeHasError,
       inputHelperText: displayHouseholdMemberAgeHelperText,
     };
 
     if (personIndex === 1) {
       return (
-        <>
+        <Box sx={{ margin: '1rem 0 1rem 0' }}>
```

Comment:
CSS trick:
```suggestion
        <Box sx={{ margin: '1rem 0' }}>
```
If you only add 2 arguments to `margin` or `padding` the first argument is the top/bottom, and the second is left/right.